src/screens/AboutScreen.js

```diff
@@ -62,15 +62,18 @@ export const AboutScreen = ({ navigation }) => {
           }
 
           let publicJsonFileContent =
-              data && data.publicJsonFile && JSON.parse(data.publicJsonFile.content);
+            data && data.publicJsonFile && JSON.parse(data.publicJsonFile.content);
 
           if (!publicJsonFileContent || !publicJsonFileContent.length) return <VersionNumber />;
 
+          // TODO accessibility not working on Title
           return (
             <ScrollView>
               {!!headlineAbout && (
-                <TitleContainer>
-                  <Title>{headlineAbout}</Title>
+                <TitleContainer accessible={true}>
+                  <Title accessibilityHint="Kopfzeile der Über die App Liste">
+                    {headlineAbout}
```

Comment:
maybe for the title we add "headline (Überschrift) (Taste)"? with the addition of "Überschrift" for headlines?

Reply:
despite the accessibility method this element remains "invisible" to VoiceOver, this same logic worked in VersionNumber, even though VersionNumber does not have the onPress logic exactly like this title component

Reply:
thanks. strange, yes. i have no idea, yet. let's have a check together next week.

Reply:
"The accessibility label on a component with accessible enabled is constructed by concatenating all children" 

from : https://sophieau.com/article/react-native-a11y/

Reply:
works from a newer test today